services/ui-src/src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import ReactDOM from "react-dom";
 import { BrowserRouter as Router } from "react-router-dom";
-import { ReactQueryDevtools } from "react-query/devtools";
+// import { ReactQueryDevtools } from "react-query/devtools";
```

Comment:
Did you mean to check in the commented code

Reply:
Haha yep I did. I'm guessing the ReactQuery dev tools will be nice to use down the road in local dev, but if it's in the DOM like this it drops an icon in the bottom left that prevents some of our E2E testing from working locally. I suppose we could use a local env variable to flip it off if running E2E tests though.